lerna.json

```diff
@@ -4,7 +4,7 @@
   ],
   "command": {
     "version": {
-      "message": "[Version Bump]: publish %s\nlerna.json\nCHANGELOG.md\n**/CHANGELOG.md"
+      "message": "[Version Bump]: ui-kit publish\nlerna.json\nCHANGELOG.md\n**/CHANGELOG.md"
```

Comment:
I removed the %s because it is [not supported in independent mode](https://github.com/lerna/lerna/blob/main/commands/version/README.md#--message-msg).

> Note that this placeholder interpolation only applies when using the default "fixed" versioning mode, as there is no "global" version to interpolate when versioning independently.